pylmod/gradebook.py

```diff
@@ -915,8 +942,50 @@ def _spreadsheet2gradebook_multi(
                         name = field
                         shortname = field[0:3] + field[-2:]
                         log.info('calling create_assignment from multi')
+
+                        # If the max_pts and normalize columns are present,
+                        # and use_max_points_column is True,
+                        # replace the default value for max points.
+                        max_points = 1.0  # the default
+                        if use_max_points_column:
+                            # This value means it was already normalized, and
+                            # we should use the default max points
+                            # instead of the one in the CSV.
+                            normalize_value = True
+                            normalize_value_str = row.get(normalize_column)
+                            if normalize_value_str is not None:
```

Comment:
There's a `log.warning` that I only want to run if something went wrong in parsing the string.